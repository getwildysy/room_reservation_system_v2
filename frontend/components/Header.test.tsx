import React from "react";
import { render } from "@testing-library/react";
import Header from "./Header";

describe("Header Component", () => {
  it("應該符合快照 (snapshot)", () => {
    // 渲染元件
    const { asFragment } = render(<Header />);

    // 產生快照並進行比對
    expect(asFragment()).toMatchSnapshot();
  });
});
