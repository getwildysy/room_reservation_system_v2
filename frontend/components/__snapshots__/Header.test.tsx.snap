// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Header Component 應該符合快照 (snapshot) 1`] = `
<DocumentFragment>
  <header
    class="bg-white shadow-sm"
  >
    <div
      class="max-w-7xl mx-auto py-4 px-4 sm:px-6 lg:px-8"
    >
      <h1
        class="text-2xl font-bold text-gray-900"
      >
        專科教室借用系統
      </h1>
    </div>
  </header>
</DocumentFragment>
`;
